randovania/games/samus_returns/logic_database/Area 6.txt

```diff
@@ -972,15 +962,12 @@ Extra - asset_id: collision_camera_060
           Any of the following:
               Space Jump or Spider Ball
               All of the following:
-                  # Reach the tunnel
+                  # Reach the Tunnel
```

Comment:
did you mean to change it from tunnel to Tunnel with a capital? I just happened to notice so just asking to be sure lol

Reply:
Oh probably not lol. Don't think I'm gonna bother fixing it right now lol

Reply:
not look it changes anything yeah. other stuff seems fine to me too

Reply:
Then I'll merge